src/pages/products/product-add/product-add-images/product-add-images.js

```diff
@@ -56,53 +51,63 @@ const ProductAddImages = ({
     if (primaryImage && additionalImages.length) {
       dispatch(setFilesToUpload([primaryImage, ...additionalImages]));
       handleNext();
-    } else if (primaryImage) {
-      dispatch(setFilesToUpload([primaryImage]));
-      handleNext();
     }
   };
 
+  const handleDeletePrimaryImage = () => {
+    setPrimaryImage('');
+  };
+
+  const handleDeleteAdditionalImage = (name) => {
+    const newImages = additionalImages.filter((image) => image.name !== name);
+    setAdditionalImages(newImages);
+  };
+
   return (
     <div className={styles.container}>
       <Box my={3}>
         <Grid container spacing={1}>
           <Grid item>
-            <div className={styles.imageUploadAvatar}>
-              <ImageUploadContainer
-                handler={handlePrimaryImageLoad}
-                buttonLabel={MAIN_PHOTO}
-              />
-              {primaryImage && (
-                <Avatar src={primaryImage}>
-                  <Image />
-                </Avatar>
-              )}
-            </div>
+            <UploadButton
```

Comment:
Please return `ImageUploadContainer` here and remove this `UploadButton` component,